pom.xml

```
@@ -4,15 +4,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.21</version>
+        <version>3.50</version>
         <relativePath />
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-scm-step</artifactId>
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Pipeline: SCM Step</name>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+SCM+Step+Plugin</url>
+    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+SCM+Step+Plugin</url>
```

This seems like a great excuse to change from using the inadequate documentation on the wiki to the existing documentation in the README.  The README is easier to read, contains more information, and better supports the user.
```suggestion
    <url>https://github.com/jenkinsci/workflow-scm-step-plugin</url>
```

@dwnusbaum this does not provide a changelog, but I suspect you will want to transition to use release drafter to provide the release notes automatically.

---

Yes, I have been migrating the changelog/documentation for Pipeline plugins to GitHub whenever I need to release a plugin that has not yet been migrated.